### PR TITLE
updated cheating response

### PR DESCRIPTION
**Pull request by chibnguyen**
Updated the cheating response to include support for the individual hackathon's rules

### CODE REVIEW

**Comment by chibnguyen** [comment]
@iaora also looking for review on this as well.

File: cheating-response.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 # MLH Cheating Response Procedure
 
-The following is a guide of actions to be taken in the case of an accusation that a team cheated or otherwise violated the rules of competition. Violations of the Code of Conduct is handled under the [Incident Response Procedure](https://github.com/MLH/policies/blob/master/incident-response.md).
+The following is a guide of actions to be taken in the case of an accusation that a team cheated or otherwise violated the rules of competition. Other violations related to the [Code of Conduct] (https://static.mlh.io/docs/mlh-code-of-conduct.pdf) is handled under the [Incident Response Procedure](https://github.com/MLH/policies/blob/master/incident-response.md).  
+
 Accusations of cheating may include but are not limited to:
 
-- A team using somebody else’s code
+- A team replicating somebody else’s code without providing proper credit
 - A team misrepresenting the work they did
```
Comment (iaora) [inline cheating-response.md]: A team misrepresenting the work that they did at the hackathon

File: cheating-response.md [hunk 2]
```diff
@@ -36,14 +37,14 @@ Once something is reported to a staff member, immediately contact the League Tea
 - Rob Kleiman - rob@mlh.io
 - Jamie Liao - jamie.liao@mlh.io
 
-You can contact them on slack (where you can also find their cell numbers). The main objectives of the meeting with the League Team are to find out the following:
+The main objectives of the meeting with the League Team are to find out the following:
```
Comment (iaora) [inline cheating-response.md]: Change wording to not have "League Team". Should be "Cheating Incidents Team"

File: cheating-response.md [hunk 3]
```diff
@@ -36,14 +37,14 @@ Once something is reported to a staff member, immediately contact the League Tea
...
 
 - What happened?
 - Are we doing anything about it?
 - Who is doing those things?
 - When are they doing them?
 
-Once League Team have been contacted they will work with the staff member to loop in the event organizer if they are not already involved.
+Once League Team have been contacted they will work with the MLH rep to loop in the event organizer if they are not already involved.
```
Comment (iaora) [inline cheating-response.md]: Don't want to be implied that it's on-site support. Generalize to generic cheating response team. Evaluate when/where want to look into the cheating incident (likely post-event)

**Comment by chibnguyen** [comment]
@theycallmeswift can you also review the updated cheating response?

**Pull request closed without merging**